apps/explorer/src/app/page.tsx

```diff
@@ -83,11 +85,38 @@ export default function Home() {
     }
   }, [uniqueTransfers, transferEvents, transactionStatusResponse?.state]);
 
+  const getUniqueAssetsBySymbol = (assets: ClientAsset[] = []) => {
+    const seen = new Map<string, ClientAsset>();
+    
+    assets.forEach(asset => {
+      const chainId = asset.chainId;
+      if (chainId && !seen.has(chainId)) {
+        seen.set(chainId, asset);
+      }
+    });
+    
+    return Array.from(seen.values());
+  };
+
   return (
     <Column gap={10}>
       <ToggleThemeButton />
 
       <Row justify="center" gap={10} >
+        <button onClick={() => {
+          NiceModal.show(Modals.AssetAndChainSelectorModal, {
+            context: "source",
+            onSelect: (asset: ClientAsset | null) => {
+              console.log("chain id selected:", asset?.chainId);
+              NiceModal.hide(Modals.AssetAndChainSelectorModal);
+            },
+            overrideSelectedGroup: {
+              assets: getUniqueAssetsBySymbol(assets),
```

Comment:
don't really like this workaround, I was thinking it would be a flag and getting the items from chains response